.gitignore

```diff
@@ -2,6 +2,7 @@
 /phpactor.schema.json
 /lib/Extension/WorseReflection/stubs
 /.phpbench
+.nova
```

Comment:
please remove this and consider using a [global git ignore file](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/) :slightly_smiling_face: